tests/fixture/testnet/local/network.go

```diff
@@ -409,14 +409,14 @@ func (ln *LocalNetwork) WaitForHealthy(ctx context.Context, w io.Writer) error {
 
 // Retrieve API URIs for all nodes in the network. Assumes nodes have
 // been loaded.
-func (ln *LocalNetwork) GetURIs() []string {
-	uris := make([]string, 0, len(ln.Nodes))
+func (ln *LocalNetwork) GetURIs() map[ids.NodeID]string {
+	uris := make(map[ids.NodeID]string, len(ln.Nodes))
 	for _, node := range ln.Nodes {
 		// Only append URIs that are not empty. A node may have an
```

Comment:
idk exactly what "loaded" in this method's comment means, but is it an issue that we may not return a map that contains all of the nodes in this network?